### PR TITLE
Add CreateStandardMultisigAccount

### PR DESCRIPTION
**Pull request by shargon**
Close https://github.com/neo-project/neo/issues/2342

### CODE REVIEW

File: src/neo/VM/Helper.cs [hunk 1]
```diff
@@ -53,17 +53,7 @@ public static ScriptBuilder EmitDynamicCall(this ScriptBuilder sb, UInt160 scrip
 
         public static ScriptBuilder EmitDynamicCall(this ScriptBuilder sb, UInt160 scriptHash, string operation, CallFlags flags, params object[] args)
         {
-            if (args.Length == 0)
```
Comment (vncoelho) [inline src/neo/VM/Helper.cs:56]: In this related to this PR, @shargon?

File: tests/neo.UnitTests/SmartContract/UT_ApplicationEngine.Contract.cs [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+                    settings.StandbyCommittee[1].EncodePoint(true),
+                    settings.StandbyCommittee[2].EncodePoint(true)
+                }
+            });
+            engine.LoadScript(script.ToArray());
+
+            Assert.AreEqual(VMState.HALT, engine.Execute());
+
+            var result = engine.ResultStack.Pop();
+            new UInt160(result.GetSpan()).Should().Be(Contract.CreateMultiSigRedeemScript(2, settings.StandbyCommittee.Take(3).ToArray()).ToScriptHash());
```
Comment (vncoelho) [inline tests/neo.UnitTests/SmartContract/UT_ApplicationEngine.Contract.cs:46]: very good!

File: src/neo/VM/Helper.cs (hunk 1 above)
Comment (shargon) [reply src/neo/VM/Helper.cs:56]: An optimization, reusing `EmitPush`

Comment (erikzhang) [reply src/neo/VM/Helper.cs:56]: I think this is not used in other place.

Comment (shargon) [reply src/neo/VM/Helper.cs:56]: If we want to call to this new syscall we will needed

**Review by erikzhang — DISMISSED** [review]

**Review by erikzhang — APPROVED** [review]

**Pull request merged**